### DISCUSSION TITLE
Error: Cannot create a string longer than 0x1fffffe8 characters

### DISCUSSION THREAD

**Discussion by hmennen90** [spatie/browsershot#642] [Q&A]
I've an Array of HTML Contents. For normal, an Element contains up to 10 HTML Pages.

But it seems to reduce the Limit in some Cases to 1 Page per HTML File (could be about 1000 and more Pages - depending to the customer).

Is it possible to implement Chunks or is there another way to work with large HTML Contents?

**Comment by huelsgp27** [comment]
some issue.

vendor/spatie/browsershot/bin/browser.js:52

```js
    output = await page[request.action](request.options);

    return output.toString('base64');
};
```
